atom/app/atom_content_client.cc

```diff
@@ -51,7 +51,6 @@
 #if defined(WIDEVINE_CDM_AVAILABLE) && BUILDFLAG(ENABLE_LIBRARY_CDMS) && \
     !defined(WIDEVINE_CDM_IS_COMPONENT)
 #define WIDEVINE_CDM_AVAILABLE_NOT_COMPONENT
```

Comment:
we can also remove the `#define WIDEVINE_CDM_AVAILABLE_NOT_COMPONENT`

Reply:
Fixed in 9a35800397906260e1c3bd58559a1bc1e74e196d.